collections/_article/dev-snapshot-godot-4-2-beta-1.md

```diff
@@ -0,0 +1,346 @@
+---
+title: "Dev snapshot: Godot 4.2 beta 1"
+excerpt: ""
+categories: ["pre-release"]
+author: Yuri Sizov
+image: /storage/blog/covers/dev-snapshot-godot-4-2-beta-1.webp
+image_caption_title: Lush Jungle
+image_caption_description: A demo scene by Wrobot
+date: 2023-10-12 14:00:00
+---
+
+Over the course of [the last 3 months](/article/godot-4-1-is-here) **270 contributors** submitted more than **1200 improvements** for the Godot 4.2 release!
+
+Among these changes are both necessary bugfixes and exciting new features, turning Godot 4 into an even better and more polished tool for you to realize your game and app ideas. There are many more improvements to come, but for now we draw the line here and turn our efforts towards stabilizing the engine so it can be released officially at the start of November.
+
+This means, _it's beta testing time,_ and you are invited to offer your hand in squashing bugs and identifying usability issues. Help us make sure that in Godot 4.2 there are no crashes, unexpected regressions, and performance is no worse (but preferably better) than before. Godot is developed by its community, and it's thanks to people like you we can be sure in the quality of our collective work!
+
+Please, consider [supporting the project financially](#support), if you are able. Godot is maintained by the efforts of volunteers and your donations go towards sponsoring their work and ensuring they can dedicate their undivided attention to the needs of the project.
+
+[Jump to the **Downloads** section](#downloads), and give it a spin right now, or continue reading to learn more about improvements in this release. You can also [try the **Web editor**](https://editor.godotengine.org/releases/4.2.beta1/) or the **Android editor** for this release. If you are interested in the latter, please request to join [our testing group](https://groups.google.com/g/godot-testers) to get access to pre-release builds.
+
+-----
+
+*The illustration picture in this article is taken from the* [**Lush Jungle**](https://wrobot.itch.io/jungledemo), *a Godot 4 demo scene created by [Wrobot](https://wrobot.nl.eu.org/). You can look at their beautify jungle vistas in [this showcase video](https://www.youtube.com/watch?v=MM_caxjlNao), or try the demo for yourself by downloading it from [itch.io](https://wrobot.itch.io/jungledemo). And consider following Wrobot on social networks ([Twitter](https://twitter.com/wrobot123), [Mastodon](https://mastodon.gamedev.place/@Wrobot)) to make sure you don't miss any of their future work!*
```

Comment:
Is the Wrobot project actually called "Lush Jungle"? The itch.io page calls it "Jungle Demo", and the Youtube video calls it "Realistic Jungle Demo".

Reply:
It's indeed called just "Jungle" but they refer to it as lush jungle in the description, which fits better our formatting than just "jungle".